Content.Server/Forensics/Systems/ForensicPadSystem.cs

```diff
@@ -0,0 +1,147 @@
+using System.Threading;
+using Content.Shared.Examine;
+using Content.Shared.Interaction;
+using Content.Shared.Inventory;
+using Content.Server.DoAfter;
+using Content.Server.Popups;
+using Robust.Shared.Player;
+
+namespace Content.Server.Forensics
+{
+    public sealed class ForensicPadSystem : EntitySystem
```

Comment:
comment, e.g. used to transfer fingerprints from target entities onto forensic pads.